### ISSUE TITLE
Quota exceeded for pagespeed

### ISSUE THREAD

**Issue by TimDaub** [JakePartusch/psi-action#5]
```
Run jakepartusch/psi-action@v1
  with:
    url: https://rugpullindex.com
    threshold: 70
    strategy: mobile
Running Page Speed Insights for https://rugpullindex.com
Error: Quota exceeded for quota metric 'Queries' and limit 'Queries per minute' of service 'pagespeedonline.googleapis.com' for consumer '<nr>'.
```

What can I do to work around this issue?

**Comment by JakePartusch** [comment]
@TimDaub — good question, I've run into a similar issue before.  I wonder if this is because we are all sharing IPs (from GitHub Actions)?

**Comment by TimDaub** [comment]
My initial reaction was that I maybe could use an API key from pagespeed to circumvent the issue. Their [docs] talk about creating a key.

Can I use a PageSpeed Insights API key with your action?

**Comment by JakePartusch** [comment]
@TimDaub, good find! It looks like it should be possible to add the key, but it is not currently supported by this action. 

If you'd like to take a stab at implementing, that would be great! Otherwise, we can leave this issue open and I'll get around to it soon enough.

Thanks!

**Comment by mohamed7320** [comment]
can you plz tell me the step by step commands to run this tool on windows command prompt i just cloned i dont know how to run the tool

**Comment by JakePartusch** [comment]
Fixed by #6
